jekyll/_cci2/jobs-steps.adoc

```diff
@@ -72,6 +72,14 @@ The `checkout` key is required under `steps` to checkout your code. The `run` ke
 
 In addition to the `run` key, keys for `save_cache`, `restore_cache`, `store_artifacts`, `store_test_results`, and `add_ssh_keys` are nested under steps. For a full list of step options see the xref:configuration-reference#steps[Steps key] section of the Configuration Reference.
 
+== Access jobs and steps in the CircleCI app
```

Comment:
I've just moved this section from the pipelines overview onto the Jobs and Steps page